pkg/dashboard/assets/css/main.css

```diff
@@ -529,3 +529,84 @@ UTILITIES
   white-space: nowrap;
   width: 1px;
 }
+
+.email-box {
+  background: #4c415f;
+  padding: 1.375rem 0.8125rem;
+  width: 100%;
+}
+
+.email-box__toggle-title {
+  color: #fff;
+  font-size: var(--step-4);
+  font-weight: 600;
+}
+
+.email-box__content p {
+  color: #fff;
+  padding: 0.75rem 0 1.5rem 0;
+}
+
+.email-box__email-label {
+  display: block;
+  position: relative;
+}
+
+.email-box__email-label span {
+  color: #575757;
+  font-weight: 400;
+  left: 0;
+  left: 0.5rem;
+  position: absolute;
+  top: 14%;
+}
+
+.email-box__email-input {
+  background: #fff;
+  border-radius: 0.25rem;
+  border: 1px solid #d3d3d3;
+  margin-bottom: 1rem;
+  padding: 0.5rem;
+  width: 71%;
+}
+
+.email-box__email-input:focus-visible {
+  outline: thick solid #4d90fe;
+}
+
+.email-box__checkbox-label {
+  color: #fff;
+  display: inline;
+  font-size: 1rem;
+}
+
+#email-box__checkbox {
+  accent-color: #dae0ec;
+}
+
+.email-box__privacy-wrapper {
+  color: #fff;
+  padding-bottom: 0 !important;
+}
+
+.email-box__privacy {
+  color: #4d90fe !important;
```

Comment:
A quick check of the contrast of this color against the purple background (again using the [WebAIM: Contrast Checker](https://webaim.org/resources/contrastchecker/)) shows that the contrast is not quite high enough (3.02:1 when it should be at least 4.5:1). Could we please adjust this color a bit?

Also, it would probably be helpful to bring the `text-decoration: underline` back in, to make it even clearer that this is a link. (I think I'm the one who universally removed the link underlines a couple years ago, which was not a great choice. My apologies!)

Reply:
I updated the code. Thank you for your feedback!

Reply:
Looks great now! Thank you!